### PR TITLE
add The Galleon

### PR DESCRIPTION
**Pull request by swiftrax**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/swiftrax/the_galleon/rules.mk [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = no         # Rotary Encoder
+OLED_DRIVER_ENABLE = yes    # I2C OLED
+WPM_ENABLE = yes            # Words per minute
```
Comment (zvecr) [inline keyboards/handwired/swiftrax/the_galleon/rules.mk:25]: Shouldnt be enabled at the keyboard layer
```suggestion
```

File: keyboards/handwired/swiftrax/the_galleon/info.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "The Galleon", 
+    "url": "github.com/swiftrax", 
+    "maintainer": "swiftrax", 
+    "width": 19.5, 
+    "height": 7.5, 
+    "layouts": {
+        "LAYOUT": {
```
Comment (zvecr) [inline keyboards/handwired/swiftrax/the_galleon/info.json:8]:
```suggestion
        "LAYOUT_all": {
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/handwired/swiftrax/the_galleon/info.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
..."y":4.25}, {"label":"10,0", "x":0, "y":5.25, "w":2.25}, {"label":"11,0", "x":2.25, "y":5.25}, {"label":"10,1", "x":3.25, "y":5.25}, {"label":"11,1", "x":4.25, "y":5.25}, {"label":"10,2", "x":5.25, "y":5.25}, {"label":"11,2", "x":6.25, "y":5.25}, {"label":"10,3", "x":7.25, "y":5.25}, {"label":"11,3", "x":8.25, "y":5.25}, {"label":"10,4", "x":9.25, "y":5.25}, {"label":"11,4", "x":10.25, "y":5.25}, {"label":"10,5", "x":11.25, "y":5.25}, {"label":"10,6", "x":12.25, "y":5.25, "w":1.75}, {"label":"10,7", "x":15.5, "y":5.25}, {"label":"11,7", "x":16.5, "y":5.25}, {"label":"10,8", "x":17.5, "y":5.25}, {"label":"11,8", "x":18.5, "y":5.25}, {"label":"11,6", "x":14.25, "y":5.5}, {"label":"12,0", "x":0, "y":6.25, "w":1.25}, {"label":"13,0", "x":1.25, "y":6.25, "w":1.25}, {"label":"12,1", "x":2.5, "y":6.25, "w":1.25}, {"label":"12,2", "x":3.75, "y":6.25, "w":6.25}, {"label":"12,4", "x":10, "y":6.25}, {"label":"13,4", "x":11, "y":6.25}, {"label":"12,5", "x":12, "y":6.25}, {"label":"13,7", "x":16.5, "y":6.25}, {"label":"12,8", "x":17.5, "y":6.25}, {"label":"13,8", "x":18.5, "y":6.25}, {"label":"12,6", "x":13.25, "y":6.5}, {"label":"13,6", "x":14.25, "y":6.5}, {"label":"12,7", "x":15.25, "y":6.5}]
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/the_galleon/info.json:9]: Better formatting, and an object order fix.

```suggestion
            "layout": [
                {"label":"0,0", "x":0, "y":0},
                {"label":"1,0", "x":1.25, "y":0},
                {"label":"0,1", "x":2.25, "y":0},
                {"label":"1,1", "x":3.25, "y":0},
                {"label":"0,2", "x":4.25, "y":0},
                {"label":"1,2", "x":5.5, "y":0},
                {"label":"0,3", "x":6.5, "y":0},
                {"label":"1,3", "x":7.5, "y":0},
                {"label":"0,4", "x":8.5, "y":0},
                {"label":"1,4", "x":9.75, "y":0},
                {"label":"0,5", "x":10.75, "y":0},
                {"label":"1,5", "x":11.75, "y":0},
                {"label":"0,6", "x":12.75, "y":0},
                {"label":"0,7", "x":15.5, "y":0},
                {"label":"1,7", "x":16.5, "y":0},
                {"label":"0,8", "x":17.5, "y":0},
                {"label":"1,8", "x":18.5, "y":0},

                {"label":"2,0", "x":0, "y":1},
                {"label":"3,0", "x":1.25, "y":1},
                {"label":"2,1", "x":2.25, "y":1},
                {"label":"3,1", "x":3.25, "y":1},
                {"label":"2,2", "x":4.25, "y":1},
                {"label":"3,2", "x":5.5, "y":1},
                {"label":"2,3", "x":6.5, "y":1},
                {"label":"3,3", "x":7.5, "y":1},
                {"label":"2,4", "x":8.5, "y":1},
                {"label":"3,4", "x":9.75, "y":1},
                {"label":"2,5", "x":10.75, "y":1},
                {"label":"3,5", "x":11.75, "y":1},
                {"label":"2,6", "x":12.75, "y":1},
                {"label":"2,7", "x":15.5, "y":1},
                {"label":"3,7", "x":16.5, "y":1},
                {"label":"2,8", "x":17.5, "y":1},
                {"label":"3,8", "x":18.5, "y":1},

                {"label":"4,0", "x":0, "y":2.25},
                {"label":"5,0", "x":1, "y":2.25},
                {"label":"4,1", "x":2, "y":2.25},
                {"label":"5,1", "x":3, "y":2.25},
                {"label":"4,2", "x":4, "y":2.25},
                {"label":"5,2", "x":5, "y":2.25},
                {"label":"4,3", "x":6, "y":2.25},
                {"label":"5,3", "x":7, "y":2.25},
                {"label":"4,4", "x":8, "y":2.25},
                {"label":"5,4", "x":9, "y":2.25},
                {"label":"4,5", "x":10, "y":2.25},
                {"label":"5,5", "x":11, "y":2.25},
                {"label":"4,6", "x":12, "y":2.25},
                {"label":"5,6", "x":13, "y":2.25},
                {"label":"9,6", "x":14, "y":2.25},
                {"label":"4,7", "x":15.5, "y":2.25},
                {"label":"5,7", "x":16.5, "y":2.25},
                {"label":"4,8", "x":17.5, "y":2.25},
                {"label":"5,8", "x":18.5, "y":2.25},

                {"label":"6,0", "x":0, "y":3.25, "w":1.5},
                {"label":"7,0", "x":1.5, "y":3.25},
                {"label":"6,1", "x":2.5, "y":3.25},
                {"label":"7,1", "x":3.5, "y":3.25},
                {"label":"6,2", "x":4.5, "y":3.25},
                {"label":"7,2", "x":5.5, "y":3.25},
                {"label":"6,3", "x":6.5, "y":3.25},
                {"label":"7,3", "x":7.5, "y":3.25},
                {"label":"6,4", "x":8.5, "y":3.25},
                {"label":"7,4", "x":9.5, "y":3.25},
                {"label":"6,5", "x":10.5, "y":3.25},
                {"label":"7,5", "x":11.5, "y":3.25},
                {"label":"6,6", "x":12.5, "y":3.25},
                {"label":"7,6", "x":13.5, "y":3.25, "w":1.5},
                {"label":"6,7", "x":15.5, "y":3.25},
                {"label":"7,7", "x":16.5, "y":3.25},
                {"label":"6,8", "x":17.5, "y":3.25},
                {"label":"7,8", "x":18.5, "y":3.25},

                {"label":"8,0", "x":0, "y":4.25, "w":1.75},
                {"label":"9,0", "x":1.75, "y":4.25},
                {"label":"8,1", "x":2.75, "y":4.25},
                {"label":"9,1", "x":3.75, "y":4.25},
                {"label":"8,2", "x":4.75, "y":4.25},
                {"label":"9,2", "x":5.75, "y":4.25},
                {"label":"8,3", "x":6.75, "y":4.25},
                {"label":"9,3", "x":7.75, "y":4.25},
                {"label":"8,4", "x":8.75, "y":4.25},
                {"label":"9,4", "x":9.75, "y":4.25},
                {"label":"8,5", "x":10.75, "y":4.25},
                {"label":"9,5", "x":11.75, "y":4.25},
                {"label":"8,6", "x":12.75, "y":4.25, "w":2.25},
                {"label":"8,7", "x":15.5, "y":4.25},
                {"label":"9,7", "x":16.5, "y":4.25},
                {"label":"8,8", "x":17.5, "y":4.25},
                {"label":"9,8", "x":18.5, "y":4.25},

                {"label":"10,0", "x":0, "y":5.25, "w":2.25},
                {"label":"11,0", "x":2.25, "y":5.25},
                {"label":"10,1", "x":3.25, "y":5.25},
                {"label":"11,1", "x":4.25, "y":5.25},
                {"label":"10,2", "x":5.25, "y":5.25},
                {"label":"11,2", "x":6.25, "y":5.25},
                {"label":"10,3", "x":7.25, "y":5.25},
                {"label":"11,3", "x":8.25, "y":5.25},
                {"label":"10,4", "x":9.25, "y":5.25},
                {"label":"11,4", "x":10.25, "y":5.25},
                {"label":"10,5", "x":11.25, "y":5.25},
                {"label":"10,6", "x":12.25, "y":5.25, "w":1.75},
                {"label":"11,6", "x":14.25, "y":5.5},
                {"label":"10,7", "x":15.5, "y":5.25},
                {"label":"11,7", "x":16.5, "y":5.25},
                {"label":"10,8", "x":17.5, "y":5.25},
                {"label":"11,8", "x":18.5, "y":5.25},

                {"label":"12,0", "x":0, "y":6.25, "w":1.25},
                {"label":"13,0", "x":1.25, "y":6.25, "w":1.25},
                {"label":"12,1", "x":2.5, "y":6.25, "w":1.25},
                {"label":"12,2", "x":3.75, "y":6.25, "w":6.25},
                {"label":"12,4", "x":10, "y":6.25},
                {"label":"13,4", "x":11, "y":6.25},
                {"label":"12,5", "x":12, "y":6.25},
                {"label":"12,6", "x":13.25, "y":6.5},
                {"label":"13,6", "x":14.25, "y":6.5},
                {"label":"12,7", "x":15.25, "y":6.5},
                {"label":"13,7", "x":16.5, "y":6.25},
                {"label":"12,8", "x":17.5, "y":6.25},
                {"label":"13,8", "x":18.5, "y":6.25}
            ]
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**